### PR TITLE
Update 12_quality_control.qmd - Grammar and punctuation corrections

### PR DESCRIPTION
**Pull request by jkc9886**
Grammatical and punctuation corrections to chapter 4. Exploration and Quality Control.

### CODE REVIEW

File: inst/pages/12_quality_control.qmd [hunk 1]
```diff
@@ -25,8 +25,8 @@ Abundance visualization is an important data exploration
 approach. `miaViz` integrated the `plotAbundanceDensity` function to plot
 the most abundant taxa along with several options.
 
-Next, a few demonstrations are shown, using the [@Lahti2014]
-dataset. A Jitter plot based on relative abundance data, similar to
+Next, a few demonstrations are shown using the [@Lahti2014]
+dataset. A jitter plot based on relative abundance data, similar to
 the one presented at [@Salosensaari2021] supplementary figure 1, can
```
Comment (antagomir) [inline inst/pages/12_quality_control.qmd]: put supplementary figure 1 in parenthese and write (Supplementary Fig. 1)

**Review by antagomir — APPROVED** [review]
Nice!

File: inst/pages/12_quality_control.qmd (hunk 1 above)
Comment (jkc9886) [reply inst/pages/12_quality_control.qmd]: Sure, done :)) Can it be merged now?

**Comment by antagomir** [comment]
@TuomasBorman will decide merge

**Review by TuomasBorman — APPROVED** [review]

**Pull request merged**